src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/X86/Avx10v1.PlatformNotSupported.cs

```diff
@@ -1678,13 +1678,13 @@ internal Avx10v1() { }
         public static Vector256<sbyte> MultiShift(Vector256<sbyte> control, Vector256<long> value) { throw new PlatformNotSupportedException(); }
 
         /// <summary>
-        ///   <para>__m256i _mm256_permutevar16x16_epi16 (__m256i a, __m256i b)</para>
+        ///   <para>__m256i _mm256_permutexvar_epi16 (__m256i idx, __m256i a)</para>
```

Comment:
The docs for these need a `<remarks>` added that the order of the parameters for native (`idx, a`) is swapped compared to the managed order (`a, idx`)

Reply:
Later this week.